src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -87,23 +109,36 @@ export const ImportBlueprintModal: React.FunctionComponent<
             browseButtonText="Upload"
             dropzoneProps={{
               accept: { 'text/json': ['.json'] },
-              maxSize: 1024,
+              maxSize: 25000,
               onDropRejected: handleFileRejected,
             }}
-            validated={isRejected ? 'error' : 'default'}
+            validated={isRejected || isInvalidFormat ? 'error' : 'default'}
           />
           <FormHelperText>
             <HelperText>
               <HelperTextItem variant={isRejected ? 'error' : 'default'}>
                 {isRejected
-                  ? 'Must be a JSON file no larger than 1 KB'
+                  ? 'Must be a valid Blueprint JSON file no larger than 25 KB'
+                  : isInvalidFormat
+                  ? 'Not compatible with the blueprints format.'
                   : 'Upload a JSON file'}
               </HelperTextItem>
             </HelperText>
           </FormHelperText>
         </FormGroup>
         <ActionGroup>
-          <Button type="button">Review and finish</Button>
+          <Button
+            type="button"
+            isDisabled={isRejected || isInvalidFormat || !jsonContent}
+            onClick={() =>
+              navigate(resolveRelPath(`imagewizard/import`), {
+                state: { blueprint: importedBlueprint },
+              })
+            }
+            ouiaId="import-blueprint-finish"
+          >
+            Review and finish
+          </Button>
```

Comment:
When I tried to import a compose request, it didn't work which makes sense. The request is missing values that blueprint needs to have.

But the failure currently crashes the service and leaves the user stuck on the `import` path which means that even refreshing the page will not help.

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/e5853b6b-da7a-40f6-ac48-43baceacfff1)